leanpkg/leanpkg/main.lean

```diff
@@ -50,7 +50,8 @@ def make : io unit := do
 manifest ← read_manifest,
 exec_cmd {
   cmd := "lean",
-  args := ["--make"] ++ manifest.effective_path,
+  args := (match manifest.timeout with some t := ["-T", t] | none := [] end) ++
```

Comment:
Note: the `timeout` setting is not propagated from dependencies.  I don't think it should, but it is something to keep in mind.